zebra-chain/src/transparent/address.rs

```diff
@@ -215,6 +215,14 @@ impl Address {
         }
     }
 
+    /// Create an address for the given public key hash and network.
+    pub fn from_public_key_hash(network: Network, pub_key_hash: [u8; 20]) -> Self {
+        Self::PayToPublicKeyHash {
+            network,
+            pub_key_hash,
+        }
+    }
+
```

Comment:
```suggestion
```

Reply:
Good catch!

Since this is consensus-critical code, we can't have two implementations, even if they are identical.

Reply:
Thank you for that, i created the method for a test and later realized we already had it. It seems i used the correct one in the test but forgot to delete the created one. Thanks again.